integrations/sdks/stainless.mdx

```diff
@@ -1,52 +1,32 @@
 ---
 title: Stainless
-description: Automate your SDK example snippets in the API playground
+description: Automate SDK example snippets in your API playground
 ---
 
-If you use Mintlify's OpenAPI support for your API reference documentation, add the following to your Stainless config:
-
-```yaml openapi.stainless.yml
-openapi:
-  code_samples: mintlify
-```
-
-Configure the [OpenAPI setup](/api-playground/openapi/setup#in-the-repo) in your Mintlify docs. To integrate Stainless, modify the GitHub Action that uploads your OpenAPI spec to Stainless so that it pushes the Stainless-enhanced OpenAPI spec into your docs repo like so:
-
-
-```yaml
-name: Upload OpenAPI spec to Stainless and (Mintlify) docs repo
-
-on:
-  push:
-    branches: [main]
-  workflow_dispatch:
-
-jobs:
-  stainless:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Push spec and config to Stainless and outputs documented spec
-        uses: stainless-api/upload-openapi-spec-action@main
-        with:
-          input_path: 'path/to/my-company-openapi.json'
-          config_path: 'path/to/my-company.stainless.yaml'
-          output_path: 'path/to/my-company-openapi.documented.json'
-          project_name: 'my-stainless-project'
-      - name: Push documented spec to docs repo
-        uses: dmnemec/copy_file_to_another_repo_action@main
-        env:
-          API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
-        with:
-          source_file: 'config/my-company-openapi.documented.json'
-          destination_repo: '{DOCS_REPO_NAME}'
-          destination_folder: 'openapi-specs' # (optional) the folder in the destination repository to place the file in, if not the root directory
-          user_email: '{EMAIL}' # the email associated with the GH token
-          user_name: '{USERNAME}' # the username associated with the GH token
-          commit_message: 'Auto-updates from Stainless'
-```
-
-This assumes the following secrets have been [uploaded to your GitHub Actions Secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions):
-
-- `secrets.STAINLESS_API_KEY`: Your Stainless API key.
-- `secrets.API_TOKEN_GITHUB`: A GitHub [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) with permissions to push to your docs repo.
+## Prerequisites
+
+* Have a valid OpenAPI specification in your `docs.json`. See [OpenAPI Setup](/api-playground/openapi/setup) for more details.
```

Comment:
I dont think this is a valid prerequisite because after you go through these steps it'll essentially generate a OpenAPI specification for you.